app/ldap/server/sync.js

```diff
@@ -274,13 +433,13 @@ export function addLdapUser(ldapUser, username, password) {
 	};
 }
 
-export function importNewUsers(ldap) {
+export function importNewUsers() {
 	if (settings.get('LDAP_Enable') !== true) {
 		logger.error('Can\'t run LDAP Import, LDAP is disabled');
 		return;
 	}
 
-	if (!ldap) {
+	if (!ldap.connected) {
 		ldap = new LDAP();
```

Comment:
Don't need to instantiate the LDAP again

Reply:
I can't remember completely, but I think I had to do this in testing because sometimes ldap went away and needed to reconnect. It wasn't always there, which is why I added this check for `ldap.connected`. I can remove it.

Reply:
Would be better to not have a global instance of LDAP and pass this to the required methods